middleware.go

```diff
@@ -16,7 +16,9 @@ type Options struct {
 	ErrHandler func(r *http.Request, w http.ResponseWriter, err error)
 }
 
-func Session(auth *jwtauth.JWTAuth, o *Options) func(next http.Handler) http.Handler {
+func Session(jwtSecret string, o *Options) func(next http.Handler) http.Handler {
```

Comment:
I'm thinking of moving jwtSecret into the options too .. and perhaps in the future we could support multiple secrets (old an new)

Reply:
This change locks us into symmetric encryption. Are we okay with that?

Reply:
another reason to move the `secret` into Options :)
 
 and then it won't lock us in 👍